mentions/views.py

```diff
@@ -7,7 +7,10 @@
 
 
 def autocomplete(request):
-    term = request.GET.get('term', '')
+    # to be haystack+solr friendly we must strip and join spaces
+    term = request.GET.get('term', '').strip()
+    term = ' '.join(term.split())
```

Comment:
How does that change the string? For me it seems that single call to `strip()` is enough.